File: src/php/db-set.php
<?php
declare(strict_types = 1);

$db = new mysqli('p:localhost', 'loadtesting', 'loadtesting', 'loadtesting', 3306, '/var/run/mysqld/mysqld.sock');
if ($db->connect_errno) {
    http_response_code(500);
    die();
}

$dbq = $db->query('INSERT INTO `table2` (col1) VALUES (1)');
if (!$dbq) {
    http_response_code(500);
    die();
}

File: src/resty/db-set.lua
local mysql = require "resty.mysql"

local db = mysql:new()

local ok = db:connect({
    path = '/var/run/mysqld/mysqld.sock',
    database = 'loadtesting',
    user = 'loadtesting',
    password = 'loadtesting'
})
if not ok then
    ngx.exit(500)
end

local res = db:query('INSERT INTO `table2` (col1) VALUES (1)')
if not res then
    ngx.exit(500)
end

local ok = db:set_keepalive(5000, 1000)
if not ok then
    ngx.exit(500)
end
